migration-engine/connectors/mongodb-migration-connector/src/lib.rs

```diff
@@ -90,9 +90,9 @@ impl MigrationConnector for MongoDbMigrationConnector {
     }
 
     async fn create_database(&self) -> ConnectorResult<String> {
-        Err(ConnectorError::from_msg(
-            "create_database() is not supported on mongodb: databases are created automatically when used.".to_owned(),
-        ))
+        let name = self.client().await?.db_name();
+        tracing::warn!("MongoDB database will be created on first use.");
```

Comment:
I think at that point the db was already created on line 93

Reply:
If the database is already created, then the warning is not necessary.

Could you check the behavior?

Reply:
No. It was not created, that call does IO but does not create a database. I warn after the call, if that IO triggers an error.